index.d.ts

```diff
@@ -0,0 +1,21 @@
+export as namespace ClientMonitor;
```

Comment:
Why don't you add a license header here?

Reply:
its rabbit write，i will delete it

Reply:
sorry,i dont know how to do right, i try it

Reply:
The tool checks you should have the Apache 2.0 header on the top of the source codes. Like other codes in the repository. You should not exclude it.

Reply:
so,I have completed the license。